kolibri/plugins/pwa/urls.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2023 Endless OS Foundation, LLC
+
+from django.conf.urls import url
+
+from .views import PwaManifestView
+
+urlpatterns = [
+    url(r"^manifest.json$", PwaManifestView.as_view(), name="manifest"),
```

Comment:
I think we should just be using one of these, not both? The shared `name` property between these two urls will cause issues as well.

It seems that `.webmanifest` is the preferred extension:

```
NOTE: File extension: .webmanifest or .json?
The IANA [registered](https://www.iana.org/assignments/media-types/application/manifest+json) file extension for the manifest is .webmanifest. Some web servers recognize this extension and transfer the file using the standardized [application manifest media type](https://www.w3.org/TR/appmanifest/#dfn-application-manifest-media-type) ([application/manifest+json](https://www.w3.org/TR/appmanifest/#dfn-application-manifest-json)). Developers can also choose a different extension (e.g. .json) or none at all (e.g. /api/GetManifest), but are encouraged to transfer the manifest using the [application/manifest+json](https://www.w3.org/TR/appmanifest/#dfn-application-manifest-json) [MIME type](https://mimesniff.spec.whatwg.org/#mime-type), although any [JSON MIME type](https://mimesniff.spec.whatwg.org/#json-mime-type) is ok.
```

And as you are setting the content_type on the template view, this should be OK?